draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -858,8 +858,10 @@ results in concatenating MP_JOIN + MP_HMAC_1 + MP_ADDADDR + MP_HMAC_2, whereas t
 first MP_HMAC_1 is associated with the MP_JOIN and the second MP_HMAC_2 is associated with the
 MP_ADDADDR suboption.
 
-If the HMAC cannot be validated by a receiving host, the subsequent handling depends
-on which suboption was being authenticated. If the suboption to be authenticated was either
+On the receiver side, the HMAC validation of the sub-options MUST be carried out according to
+the sending sequence in which the associated MP_HMAC follows a sub-option. If the suboption
+cannot be validated by a receiving host because the HMAC validation fails, the subsequent handling depends
+on which suboption was being verified. If the suboption to be authenticated was either
```

Comment:
And if it does not follow a suboption, do you ignore the HMAC or drop the subflow/connection? Would be good to clarify.

We also said we should clarify the wording around the three suboptions, but maybe this is a different issue/PR?

Reply:
@abrunstrom, how to handle non valid HMAC is described in the last paragraph of the MP_HMAC section.

Reply:
@markusa not sure what text you are referring to? Did you mean somewhere else? This is the last paragraph of the MP_HMAC section.

Reply:
@abrunstrom  Please read https://github.com/tsvwg/ietf-multipath-dccp/pull/279/commits/b98e1c90e64749c08bd89eeed0f38213eff6e6c7 which I think addresses your comment

The second part of your comment will be handled by @veselinrak in the course of the MP_HMAC related Issue #245